department-of-veterans-affairs/caseflow
# HunJerBAH/APPEALS-17551

Resolves [APPEALS-17551](https://vajira.max.gov/browse/APPEALS-17551)

# Description
Created post function on intake to create Request Issues with MST and PACT status from the Intake User's selection.

## Acceptance Criteria
- [ ] When the intake process is complete, issues that have MST check create Request Issues with MST.
- [ ] When the intake process is complete, issues that have PACT check create Request Issues with PACT.

## Testing Plan

1. Go to Intake as a BVA intake user (BVADWISE).
2. Select Decision Review from the dropdown.
3. Enter a veteran file (I used 400000004.
4. Fill out the form.
5. Click the add Issues button.
6. Select an old issue and the MST/PACT checkboxes, or create a new issue and select the MST/PACT checkboxes. Click **save** to save the issue. 
7. Click **establish appeal**
![image](https://user-images.githubusercontent.com/99915461/235197332-160c4828-d9c9-43ec-8b29-5bdf1b6c14a5.png)
9. Intake should show success. Click the **correct issues** link.
10. The MST and PACT status should show in the Redux store and on the Request Issue model.
![image](https://user-images.githubusercontent.com/99915461/235198397-fab5a33f-fefc-4e46-9033-fa8009938e0f.png)
![image](https://user-images.githubusercontent.com/99915461/235198759-94569738-fa49-42ac-8e73-feff1faab3ea.png)
